web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknad/SøknadRoutes.kt

```diff
@@ -40,6 +41,11 @@ internal fun Route.søknadRoutes(
     søknadService: SøknadService,
     lukkSøknadService: LukkSøknadService,
 ) {
+    authorize(Brukerrolle.Veileder) {
```

Comment:
Selve søknadsendepunktet har disse brukerrollene:
>     authorize(Brukerrolle.Veileder, Brukerrolle.Saksbehandler) {
        post(søknadPath) {

Reply:
Mulig vi gjenbruker noe her for registrering av papirsøknad?